### ISSUE TITLE
Create validation set

### ISSUE THREAD

**Issue by eburch2** [johnso4/PhishingClassification#5]
Since we don't have a lot of data, we need to find a split 90/10 or 70/30 of training and test data. We should also consider LOOCV to split our data into test data and training data. Then we can solve for our train/test error given the two sets.

**Comment by eburch2** [comment]
#8 and #9

**Comment by eburch2** [comment]
function for creating validation sets is complete as of 11.16.17
